doc/manual/cookbooks/sinker-with-active-particles/sinker-with-active-particles.prm

```diff
@@ -0,0 +1,108 @@
+set Dimension                              = 2
```

Comment:
In the cookbook, we typically don't include the *whole* .prm file, unless essentially everything is new. Rather, we only include those parts that really are specific to what is being demonstrated. Can you reduce what you include here to the important parts? I suspect that it is based on a .prm file that has previously been discussed in the manual anyway, and then you can say that it is "based on the parameter file discussed in Section~\ref{...}, with only the following sections changed in relevant ways (the entire input file can be found in \url{...}):"